javascript/javascript1/week3/notes.js

```diff
@@ -0,0 +1,42 @@
+console.log("================ notes ================");
+
+const notes = [];
+
+// ########################## saveNote ##########################
+
+function saveNote(content, id) {
+  const time = new Date(); // adding unique feature (adding current time for each note)
+  notes.push({ content: content, id: id, time: time.toLocaleTimeString() });
```

Comment:
You can write this a bit shorter when the key is the same as the property:
```suggestion
  notes.push({ content, id, time: time.toLocaleTimeString() });
```